architecture/3-DataStructures/1-Satellite_Health.yml

```diff
@@ -0,0 +1,44 @@
+name: Satellite Health Data Packet
+purpose: Data structure for satellite health data packet.
+template: |
+  imux: {{imux}}
+  imuy: {{imuy}}
+  imuz: {{imuz}}
+  imuTemperature: {{imuTemperature}}
+  batteryVoltage: {{batteryVoltage}}
+  batteryCurrent: {{batteryCurrent}}
+  batteryBusCurrent: {{batteryBusCurrent}}
+  batteryCurrentDirection: {{batteryCurrentDirection}}
+  batteryTemperature: {{batteryTemperature}}
+  battery12VBusCurrent: {{battery12VBusCurrent}}
+  battery5VBusCurrent: {{battery5VBusCurrent}}
+  battery3dot3VBusCurrent: {{battery3dot3VBusCurrent}}
+  transmitterCurrent: {{transmitterCurrent}}
```

Comment:
Measure of current (mA) being drawn by the radio transmitter during transmission.